packages/components/src/components/message/message.tsx

```diff
@@ -20,6 +23,7 @@ interface MessageProps extends ComponentPropsWithRef<'div'> {
 export const Message = forwardRef<ElementRef<'div'>, MessageProps>(
   ({ className, children, variant, ...props }, ref) => (
     <div className={cn(messageVariants({ variant, className }))} ref={ref} {...props}>
+      {!variant && <Info className="flex-none" />}
```

Comment:
What are your thoughts on adding a default `info` variant?

Reply:
Sure, that works!